collector/netdev_bsd.go

```diff
@@ -101,7 +102,6 @@ func getNetDevStats() (map[string]map[string]string, error) {
 			devStats["receive_multicast"] = strconv.Itoa(int(data.ifi_imcasts))
 			devStats["transmit_multicast"] = strconv.Itoa(int(data.ifi_omcasts))
 			devStats["receive_drop"] = strconv.Itoa(int(data.ifi_iqdrops))
-			devStats["transmit_drop"] = strconv.Itoa(int(data.ifi_oqdrops))
```

Comment:
I wonder if it makes sense to remove this metric from freebsd. Seems to be quite interesting?